src/one/nio/serial/GeneratedSerializer.java

```diff
@@ -53,7 +60,13 @@
         }
 
         checkFieldTypes();
+        for (FieldDescriptor fd : fds) {
+            if (!generationInProcess.contains(fd.type().resolve())) {
+                Repository.get(fd.type().resolve());
```

Comment:
Without ```generationInProcess```  I've got StacktraceException even on serialize ```NullPointerException.class``` because the Throwable contains [field with type Throwable.class](http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/6-b14/java/lang/Throwable.java#Throwable.0cause)